packages/endpoint/0.20.1/manifest.yml

```diff
@@ -0,0 +1,27 @@
+format_version: 1.0.0
+name: endpoint
+title: Endpoint Security
+description: Protect your hosts with threat prevention, detection, and deep security data visibility.
+version: 0.20.1
+categories: ["security"]
+# Options are experimental, beta, ga
+release: beta
```

Comment:
Not sure if we're moving endpoint package from beta to ga since fleet is going ga.